modules/planning/tasks/deciders/path_assessment_decider/path_assessment_decider.cc

```diff
@@ -84,8 +84,10 @@ Status PathAssessmentDecider::Process(
          << " msec.";
 
   // 2. Analyze and add important info for speed decider to use
+  size_t cnt = 0;
   const Obstacle* blocking_obstacle_on_selflane = nullptr;
-  for (auto& curr_path_data : valid_path_data) {
+  for (size_t i = 0; i != valid_path_data.size(); ++i) {
+    auto& curr_path_data = valid_path_data[i];
     if (curr_path_data.path_label().find("fallback") != std::string::npos) {
```

Comment:
the fallback path data may not be skipped for empty check added below.

Reply:
@yifeijiang  Got it. Will resolve it.

Reply:
Done.